Rules.py

```diff
@@ -200,7 +224,6 @@ def global_rules(world):
 
     set_rule(world.get_entrance('Desert Ledge Mirror Spot'), lambda state: state.has_Mirror())
     set_rule(world.get_entrance('Desert Palace Stairs Mirror Spot'), lambda state: state.has_Mirror())
-    set_rule(world.get_entrance('Desert Palace Entrance (North) Mirror Spot'), lambda state: state.has_Mirror())
```

Comment:
Why was this removed in non-inverted? I'm pretty sure we don't want people to be able to mirror from dark world to light world without having a mirror!

Reply:
Thanks for pointing this out, this was just me being careless with my editor's search.